src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -344,6 +344,9 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
   );
 };
 
+const ensureGeneViewExists = (view: string) =>
+  !!Object.values(View).find((value) => value === view);
```

Comment:
`!Object.values(View).find` is identical to `.some`

Reply:
I don't know how I ended up with `find` when I actually wanted to use `some`. Thanks for pointing that out 😀

Reply:
✅